simpleprophet/simpleprophet/output.py

```diff
@@ -83,9 +86,25 @@ def write_forecasts(bigquery_client, table, modelDate, forecast_end, data, produ
       "asofdate", "datasource", "date", "type", "value", "low90", "high90",
       "p10", "p20", "p30", "p40", "p50", "p60", "p70", "p80", "p90"
     ]]
-    output_data['date'] = pd.to_datetime(output_data['date']).dt.date
-    errors = bigquery_client.insert_rows(
+    output_data['date'] = output_data['date'].dt.strftime('%Y-%m-%d')
```

Comment:
Just want to confirm that the date format here and the isoformat call on 73 are correct since the data and asofdate fields are still DATE in the schema?

Reply:
Yeah, I will add some comments about that. For purposes of preparing a newline-delimited json payload, BigQuery expects DATE types to be ISO-formatted strings, which is why we do this conversion.